### DISCUSSION TITLE
Invalid vercel.json - `builds` should NOT have more than 128 items.

### DISCUSSION THREAD

**Discussion by ScottAgirs** [vercel/next.js#25602] [Help] [answered]
### What version of Next.js are you using?

10.0.9

### What version of Node.js are you using?

14.16.0

### What browser are you using?

Chrome, Safari, Brave

### What operating system are you using?

macOS Big Sur

### How are you deploying your application?

vercel dev

### Describe the Bug

Trying to test locally with `vercel dev` throws the following error
![image](https://user-images.githubusercontent.com/11737572/120083031-18e0a380-c094-11eb-9977-74be110b2aed.png)

I do not have a `vercel.json` initialized on my project, nor does it make any difference if I initialize `vercel.json` file with an empty object within.

### Expected Behavior

Spin up vercel deployment locally

### To Reproduce

If there is no known steps I could try I will set up a sandbox, but hoping there's something obvious that I'm missing or that is missing from docs.

**Comment by timneutkens** [comment, marked as answer]
Hey! Could you create a support case at https://vercel.com/support/request, that'll allow the customer success team to investigate the issue with you 🙏
